spring-cloud-consul-discovery/src/test/java/org/springframework/cloud/consul/discovery/ConsulDiscoveryPropertiesTests.java

```diff
@@ -19,15 +19,20 @@
 import java.util.Collections;
 import java.util.Map;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
```

Comment:
Again, here I converted over to Jupiter. I know we strive to not pull multiple levers in a single pull request. However, the changes were significant enough in this test that I went ahead and bumped up. I feel that if we don't do this a little along the way that JUnit4 will stay in here for quite some time.